static_frame/core/container_util.py

```diff
@@ -1822,14 +1815,13 @@ def prepare_values_for_lex(
     """Prepare values for lexical sorting; assumes values have already been collected in reverse order. If ascending is an element and values_for_lex is None, this function is pass through."""
     asc_is_element = isinstance(ascending, BOOL_TYPES)
     if not asc_is_element:
-        ascending = tuple(ascending)  # type: ignore
```

Comment:
This was needed before because you had support for `tp.Iterable[bool]`, but per our discussion, the signature now requires sized containers of bool